BO4E/ENUM/Status.cs

```diff
@@ -60,6 +60,39 @@ public enum Status
         /// <remarks>Z39</remarks>
         ZAEHLERSTAND_ZUM_ENDE_DER_ANGEGEBENEN_ENERGIEMENGE_NICHT_VORHANDEN_DA_MENGENABGRENZUNG,
 
+        /// <summary> KEIN_ZUGANG </summary>
+        /// <remarks>Z74</remarks>
+        KEIN_ZUGANG,
+
+        /// <summary> KOMMUNIKATIONSSTOERUNG </summary>
+        /// <remarks>Z75</remarks>
+        KOMMUNIKATIONSSTOERUNG,
+
+        /// <summary> NETZAUSFALL </summary>
+        /// <remarks>Z76</remarks>
+        NETZAUSFALL,
+
+        /// <summary> STATUS_GERAETEWECHSEL </summary>
+        /// <remarks>Needs to have a prefix, because this is also an enum value in <see cref="Themengebiet" /></remarks>
```

Comment:
Wo sehe ich eigentlich die Dokumentation die hier steht?
Gibt es da auch sowas wie readthedocs für C#?

Reply:
Weiß ich nicht^^

Reply:
Also:
1. Es gibt in diesem Repo einen Workflow: https://github.com/Hochfrequenz/BO4E-dotnet/blob/main/.github/workflows/trigger_doc_build.yml
2. Der ruft per webhook dieses Repo auf: https://github.com/Hochfrequenz/bo4e-doc/blob/d817afa32f1e693f999ae1949a2af76965520811/.github/workflows/build_docs.yml#L4
3. der workflow aus 2 buildet dann mittels docfx die doku die in https://github.com/Hochfrequenz/bo4e-livedocs liegt

Mindestes ein glied dieser workflow kette ist aktuell defekt.
Wahrscheilich geht das alles einfacher.

Aber so war mal der plan